xlators/mgmt/glusterd/src/glusterd-snapshot-utils.c

```diff
@@ -4133,6 +4133,8 @@ glusterd_snapshot_plugin_by_name(char *name,
 
     if (strcmp(name, "LVM") == 0)
         *snap_ops = &lvm_snap_ops;
+    else if (strcmp(name, "ZFS") == 0)
```

Comment:
can this be `strcasecmp()` ?

Reply:
This is comparing the plugin name that is provided here https://github.com/gluster/glusterfs/pull/2855/files#diff-e56fa01b9d9da20ebddaf48db7ad09e7d89900cf72aafd840943e26f3cc66024R513

Do we need to change this to `strcasecmp` since this is not user input or coming from external?

Reply:
In that case, this should be fine.